From a774945fe7d0c9a3ddc0c7e32ef1f25cbbaf18b6 Mon Sep 17 00:00:00 2001
From: Simon McVittie <smcv@debian.org>
Date: Mon, 16 Jan 2017 00:29:19 +0000
Subject: [PATCH 1/2] doc: Define new uses-network restriction

nss-mdns' tests should have this restriction, because it asserts that
nss-mdns-test-$(dbus-uuidgen).nonexistent.invalid doesn't exist.
---
 doc/README.package-tests.rst | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/doc/README.package-tests.rst b/doc/README.package-tests.rst
index 4775a07..3117f3d 100644
--- a/doc/README.package-tests.rst
+++ b/doc/README.package-tests.rst
@@ -210,6 +210,13 @@ needs-recommends
     Enable installation of recommended packages in apt for the test
     dependencies. This does not affect build dependencies.
 
+uses-network
+    The test might contact other machines, so is not suitable to
+    be run in an environment where privacy is essential.
+    Tests should avoid relying on remote network services, but in
+    some cases a test attempts to contact the network as a side-effect,
+    for example resolving a hostname in DNS.
+
 Defined features
 ----------------
 
-- 
2.11.0

